### PR TITLE
Update spanish po file

### PR DESCRIPTION
**Pull request by JotaRandom**
This pull update the po/es.po file.

Not other thing to say, just I update and fix the fuzzy ones, that all.

### CODE REVIEW

File: po/es.po [hunk 1]
```diff
@@ -404,19 +404,19 @@ msgstr ""
 
 #: ../src/login/org.freedesktop.login1.policy.in.h:47
 msgid "Hibernate the system while an application asked to inhibit it"
-msgstr "Hibernar el sistema a pesar de que una aplicación lo impide"
+msgstr "Hibernar el sistema a pesar de que una aplicación lo impide."
```
Comment (martinpitt) [inline po/es.po]: Please don't end translated sentences with periods if the original sentence doesn't.

**Comment by martinpitt** [comment]
One nitpick (which applies to several places), otherwise LGTM, thank you!

**Comment by JotaRandom** [comment]
I think I finally did it right, I squashed all in one commit and I think I fix all my misplaced dots

**Comment by martinpitt** [comment]
Appreciated, thank you!

**Pull request merged**